lib/chat_app_web/live/single_room_live.ex

```diff
@@ -7,23 +7,62 @@ defmodule ChatAppWeb.SingleRoomLive do
   @impl true
   def mount(%{"id" => room_id}, _session, socket) do
     topic = "room:" <> room_id
+
     if connected?(socket) do
-        Logger.info("Liveview's second mount() call")
-        LiveviewMonitor.monitor(self(), __MODULE__, %{id: socket.id, room_id: room_id})
-        PubSub.subscribe(ChatApp.PubSub, topic)
-        broadcasted_message = {:add_messages, ChatApp.Message.new("anon joined the chat.", "", room_id)}
-        PubSub.broadcast(ChatApp.PubSub, topic, broadcasted_message)
+      Logger.info("Liveview's second mount() call")
+      LiveviewMonitor.monitor(self(), __MODULE__, %{id: socket.id, room_id: room_id})
+      PubSub.subscribe(ChatApp.PubSub, topic)
+
+      message = %{
+        "content" => "anon joined the chat",
+        "sender" => "",
+        "room_id" => room_id,
+        "timestamp" => DateTime.utc_now(),
+        "is_deleted" => false,
+        "is_edited" => false,
+        "seq_number" => 0
+      }
+
+      IO.inspect(socket.assigns)
+
+      IO.inspect(message, label: "Broadcasted join message")
```

Comment:
same story as console.logs - this is related to local debugging, and it is a good practice to remove them once we finish debugging the feature we are working on